subsys/logging/log_output.c

```diff
@@ -88,7 +88,7 @@ static int timestamp_print(struct log_msg *msg,
 	int length;
 	u32_t timestamp = log_msg_timestamp_get(msg);
 
-	if (!format) {
+	if (!format || (freq == 0)) {
```

Comment:
Could we just skip timestamp field in this case?

Reply:
we can do that. When freq=0 and formating is enabled it is most probably a case when default timestamping source is used and CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=0. In that case there is no point in printing [00000000] timestamp before every log.